client/state/plugins/installed/selectors.js

```diff
@@ -52,6 +52,13 @@ export function isRequestingForSites( state, sites ) {
 	return some( sites, ( siteId ) => isRequesting( state, siteId ) );
 }
 
+export function isRequestingForAllSites( state ) {
+	if ( typeof state.plugins.installed.isRequestingAll === 'undefined' ) {
+		return false;
+	}
+	return state.plugins.installed.isRequestingAll;
```

Comment:
Is this possible? I see that this piece of state has a default value of `false`, so I don't think it can be `undefined`.

Reply:
I second this. Should be fine to just return without a check.

Reply:
Thanks! Fixed in f47988bd6bbdc03b69064032f23a2db030cb49ff